worlds/shivers/Rules.py

```diff
@@ -225,7 +228,11 @@ def set_rules(world: "ShiversWorld") -> None:
     forbid_item(multiworld.get_location("Ixupi Captured Metal", player), "Metal Always Available in Prehistoric", player)
 
     # Set completion condition
-    multiworld.completion_condition[player] = lambda state: (first_nine_ixupi_capturable(state, player) and lightning_capturable(state, player))
+    multiworld.completion_condition[player] = lambda state: ((
+                water_capturable(state, player) + wax_capturable(state, player) + ash_capturable(state, player) \
+                + oil_capturable(state, player) + cloth_capturable(state, player) + wood_capturable(state, player) \
+                + crystal_capturable(state, player) + sand_capturable(state, player) + metal_capturable(state, player) \
+                + lightning_capturable(state, player)) >= world.options.ixupi_captures_needed.value)
```

Comment:
Do you think players are more likely to play with the maximum value for ixupi_captures_needed, or closer to the minimum value?

If it's usually closer to the max value, what you're doing is better
If it's usually closer to the min value, there is a better way to do this

Basically the consideration here is that this could be a for loop with early exit instead. Right now, it calls every function and then compares at the end, when it could stop as soon as it hits the value of `world.options.ixupy_captures_needed`.

*Low Priority*

Reply:
They will either play with maximum value general. If not I expect >= 5.